forward/forward_test.go

```diff
@@ -35,7 +35,8 @@ func TestCloneRequest(t *testing.T) {
 	const rawPath = "/%E4%B8%9C%E6%96%B9Project"
```

Comment:
Yeah.  The weird thing though is when I dump the outbound requests, the http library isn't using the full URL in the request line anyway.  In any case, it works :)